hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sinks.java

```diff
@@ -51,6 +51,9 @@
  * a sink is a transform that has no output. A pipeline stage with a sink
  * as its transform has the type {@link SinkStage} and accepts no
  * downstream stages.
+ * <p>
+ * The default local parallelism for sources in this class is 1, check the
```

Comment:
Is this necessary? Since there's documentation on each method. One of the methods actually has a different default parallelism. Maybe say "The default local parallelism for _most_ of the sources in this class is 1".

Reply:
I think it's good to mention common distinctive feature of the processors here. I changed to "is typically 1".